build.zig

```diff
@@ -202,9 +203,13 @@ pub fn build(b: *std.Build) void {
         .optimize = optimize,
         .strip = false,
     });
-    blk_virt.addCSourceFile(.{ .file = b.path("blk/components/virt.c"), .flags = &.{b.fmt("-DBLK_NUM_CLIENTS={any}", .{blk_num_clients_opt})} });
-    blk_virt.addCSourceFiles(.{ .files = &.{ "blk/util/bitarray.c", "blk/util/fsmalloc.c", "blk/util/util.c", "util/cache.c" } });
+    blk_virt.addCSourceFile(.{
+        .file = b.path("blk/components/virt.c"),
+        .flags = &.{"-fno-sanitize=undefined"}
```

Comment:
The virtualiser was being compiled with the UB sanitizer on, now it's off. Ideally it should always be on. What changed?

Reply:
nothing's changed, there wasn't any runtime errors when i ran the virtio example and im just matching the fact that all the other components have it off.